src/app/globals.css

```
@@ -149,6 +149,17 @@ h6 {
 	color: var(--color-secondary);
 }
 
+@keyframes fadeSlideIn {
```

Stylelint (17.7.0)

[error] Expected keyframe name "fadeSlideIn" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)